backend/contract-service/src/main/java/org/egov/works/kafka/NotificationConsumer.java

```diff
@@ -0,0 +1,45 @@
+package org.egov.works.kafka;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.egov.works.service.NotificationService;
+import org.egov.works.web.models.ContractRequest;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.support.KafkaHeaders;
+import org.springframework.messaging.handler.annotation.Header;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+
+@Service
+@Slf4j
+public class NotificationConsumer {
+    @Autowired
+    NotificationService notificationService;
+
+    @Autowired
+    private ObjectMapper mapper;
+
+
+    /**
+     * Consumes the create contract record and send notification
+     *
+     * @param record
+     * @param topic
+     */
+
```

Comment:
You can comment this Kafka listener if not processing msg

Reply:
Kafka listner is removed